pasta_eln/database.py

```diff
@@ -43,11 +44,16 @@ def __init__(self, user:str, password:str, databaseName:str, configuration:dict[
       self.db.create_document(self.ontology)
       self.initDocTypeViews( configuration['tableColumnsMax'] )
       self.initGeneralViews()
-    self.ontology = self.db['-ontology-']
-    if '-version' not in self.ontology or self.ontology['-version']!=2:
+    self.ontology = dict(self.db['-ontology-'])
```

Comment:
Culprit is here itself :) Ontology is converted to dict, hence the save call is failing inside ontology editor!

Reply:
yes, see above for the long discussion an couchDB documents.
BTW, the new version of the library has no documents anymore, only dicts/json (as far as I understand)

Reply:
Ahh so we are planning to do an upgrade soon to the new version of couchdb client library, is that right? If this is the case then it would be a major change right?

Reply:
As per the discussion, this issue will be addressed separately as part of [Save fails in ontology editor #42](https://github.com/PASTA-ELN/pasta-eln/issues/42)